cluster/images/nginx/nginx.json

```diff
@@ -0,0 +1,48 @@
+{
+"apiVersion": "v1beta3",
+"kind": "Pod",
+"metadata": {"name":"nginx"},
+"spec":{
+"hostNetwork": true,
+"containers":[
+    {
+    "name": "nginx",
+    "image": "gcr.io/google-containers/nginx:v1",
+    "command": [
+              "nginx",
+	      "-g",
+	      "daemon off;"
+	],
+    "ports":[
+      { "name": "https",
+        "containerPort": 443,
+        "hostPort": 443}
+        ],
+    "volumeMounts": [
```

Comment:
you might want to mount /var/log/nginx as well.
check out https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/saltbase/salt/nginx/nginx.conf#L38